.github/instructions/nx.instructions.md

```diff
@@ -34,14 +34,13 @@ If the user wants help with tasks or commands (which include keywords like "test
 - If there are any tasks, ask the user if they would like help with a specific task then use the 'nx_current_running_task_output' tool to get the terminal output for that task/command
```

Comment:
@tschaffter this file is automatically changed~ should I consider removing it from this pr?

Reply:
@rrchai Yes, please remove the changes made to it in this PR.